libraries/util/profile/myBeans.py

```diff
@@ -70,4 +70,30 @@ def verifyBeansRewardFromOnRegistration(page):
         """  
         if uCommon.verifyVisible(page, pMyBeans.com.onRegistrationLbl) == True:
             uCommon.expectElemToBeVisible(page, pMyBeans.com.onRegistrationRewardLbl)
-            uCommon.log(1, f'On Registration activity is visible.') 
\ No newline at end of file
+            uCommon.log(1, f'On Registration activity is visible.') 
+            
+    @uCommon.ufuncLog   
+    def verifyOnEdamamaRewards(page, value):
+        """ 
+        Objective: To verify On Edamama Rewards.
+        param: None
+        returns: None
+        Author: abernal_20240311
+        """  
+        if uCommon.verifyVisible(page, pMyBeans.com.onEdamamaRewardsLbl) == True:
+            elemValue = uCommon.getElemText(page, pMyBeans.com.onEdamamaRewardsValueLbl)
+            if uCommon.waitAndValidateElemText(page, elemValue, value) == True:
+                uCommon.log(1, f'On Edamama Rewards(+) activity is visible.') 
```

Comment:
Please add to the condition when returned value is False =
uCommon.log(2, f'On Edamama Rewards(+) activity is NOT visible.')

so we can catch both result.